bastion/gui/dashboard/main_window.py

```
@@ -0,0 +1,882 @@
+        try:
+            data = self.sock.recv(4096).decode()
+            if not data:
+                # Disconnected
+                self.disconnect_from_daemon()
+                return
+
+            self.buffer += data
+            while '\n' in self.buffer:
+                line, self.buffer = self.buffer.split('\n', 1)
+                self.process_daemon_message(line)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)